app/src/mainwindow2.cpp

```diff
@@ -44,7 +44,11 @@ GNU General Public License for more details.
 #include "soundmanager.h"
 #include "viewmanager.h"
 
+#include "layer.h"
 #include "layercamera.h"
+#include "layerbitmap.h"
+#include "bitmapimage.h"
```

Comment:
Core files should not exist in app. Only widgets and UI related should exist here.

Reply:
Remember to remove the header files again that you don't use.